.github/workflows/build-lf-petal-keycloak.yml

```diff
@@ -0,0 +1,42 @@
+name: Build lf petal keycloak image
+
+on:
+  workflow_dispatch:
+    inputs:
+      Tag:
+        type: string
+        description: Image tag
+        required: true
+        default: latest
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_TAG: '${{ github.event.inputs.Tag }}'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up JDK 21 for x64
+        uses: actions/setup-java@v4
+        with:
+          java-version: '21'
+          distribution: 'corretto'
+          architecture: x64
+      - name: Build jar
+        run: |
+          ./mvnw -pl quarkus/deployment,quarkus/dist -am -DskipTests clean install
```

Comment:
pls use `./mvnw clean install -Pdistribution -DskipTests`, the command that you are using will not generate tar.gz package.